src/cargo_ops/mod.rs

```diff
@@ -43,6 +43,11 @@ struct Manifest {
     )]
     pub target: Option<Table>,
     pub features: Option<Value>,
+    #[serde(
```

Comment:
Isn't that a wrong file? The `[net]` section belongs to `.cargo/config`, not `Cargo.toml`

Reply:
Seems perfectly valid from the [offical documentation](https://doc.rust-lang.org/cargo/reference/config.html) (you'll have to ctrl+f for `[net]` I can't link directly to it :()

Unless I misunderstood your point?

Reply:
You've linked to documentation of `.cargo/config` file, but you've modified parsing code of `Cargo.toml` file. They're both Cargo's TOML files, but they're not the same kind of file.

Reply:
Ah, I see where I glossed over that and put it in the `Cargo.toml` instead of the `.cargo/config` whoops!  I'll get that fixed soon :)